lib/peripheral.dart

```diff
@@ -28,32 +47,57 @@ class Peripheral {
           refreshGatt: refreshGatt,
           timeout: timeout);
 
+  /// Observe connection state
+  ///
+  /// Returns stream of [PeripheralConnectionState]
   Stream<PeripheralConnectionState> observeConnectionState(
           {bool emitCurrentValue = false, bool completeOnDisconnect = false}) =>
       _manager.observePeripheralConnectionState(
           identifier, emitCurrentValue, completeOnDisconnect);
 
+  /// Check connection state of the Device.
   Future<bool> isConnected() => _manager.isPeripheralConnected(identifier);
 
+  /// Disconnects from Device if it's connected or cancels pending connection.
   Future<void> disconnectOrCancelConnection() =>
       _manager.disconnectOrCancelPeripheralConnection(identifier);
 
+  /// Discovers all Services, Characteristics and Descriptors for Device.
+  ///
+  /// Optional [transactionId] could be used to cancel operation.
   Future<void> discoverAllServicesAndCharacteristics({String transactionId}) =>
       _manager.discoverAllServicesAndCharacteristics(
           this, transactionId ?? TransactionIdGenerator.getNextId());
 
+  /// List of discovered Services for the Device.
   Future<List<Service>> services() => _manager.services(this);
 
+  /// List of discovered Characteristics for given Service.
+  ///
+  /// [servicedUuid] must be specified and characteristics only for that
+  /// service are returned.
   Future<List<Characteristic>> characteristics(String servicedUuid) =>
       _manager.characteristics(this, servicedUuid);
 
+  /// Reads RSSI for the device.
+  ///
+  /// Optional [transactionId] could be used to cancel operation.
   Future<int> rssi({String transactionId}) =>
       _manager.rssi(this, transactionId ?? TransactionIdGenerator.getNextId());
 
+  /// Request new MTU value for this device. This function currently is not
```

Comment:
```suggestion
  /// Request new MTU value for this peripheral. This function currently is not
```
Since we're in peripheral class I think it's more obvious when we use the same naming in doc.